### ISSUE TITLE
How to get extras when installing from git source?

### ISSUE THREAD

**Issue by DiagonalArg** [rndusr/stig#84]
Could you tell us how to install geoip and setproctitle when installing as:

```
pip3 install [--user] git+https://github.com/rndusr/stig.git
```

Maybe put that in the README?

Thanks - (Fantatic software!)

**Comment by DiagonalArg** [comment]
Thanks.  

And setproctitle should be used as?

```
set proctitle "TITLE"
```

I don't have the dependency, so I can't check.

**Comment by rndusr** [comment]
No, you just have to install the setproctitle module. stig should automatically
detect if it's running in a tmux session and remove any CLI arguments from the
the process title.
